condominium/data/ir_ui_view.xml

```diff
@@ -0,0 +1,461 @@
+                </xpath>
+                <xpath expr="//field[@name='x_partner_id']" position="attributes">
+                    <attribute name="options">{"no_create":true}</attribute>
+                    <attribute name="required">True</attribute>
+                    <attribute name="string">Owner</attribute>
+                </xpath>
+                <xpath expr="//field[@name='x_tag_ids']" position="before">
+                    <field name="x_area" required="True"/>
+                    <field name="x_area_percent" widget="percentage"/>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="list_view_x_unit_custom" model="ir.ui.view">
+        <field name="name">Default list view for x_units customization</field>
+        <field name="inherit_id" ref="list_view_units"/>
+        <field name="mode">extension</field>
+        <field name="model">x_units</field>
+        <field name="priority">99</field>
+        <field name="type">tree</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='x_sequence']" position="after">
+                    <field name="x_condominium_id" optional="show"/>
+                </xpath>
+                <xpath expr="//field[@name='x_name']" position="attributes">
+                    <attribute name="optional">show</attribute>
+                </xpath>
+                <xpath expr="//field[@name='x_partner_id']" position="attributes">
+                    <attribute name="optional">show</attribute>
+                </xpath>
+                <xpath expr="//field[@name='x_partner_id']" position="after">
+                    <field name="x_tenant_id" optional="hide"/>
+                </xpath>
+                <xpath expr="//field[@name='x_tag_ids']" position="attributes">
+                    <attribute name="optional">show</attribute>
+                </xpath>
+                <xpath expr="//field[@name='x_tag_ids']" position="after">
+                    <field name="x_area" optional="show"/>
+                    <field name="x_area_percent" optional="show"/>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="res_partner_form_custom" model="ir.ui.view">
+        <field name="name">res.partner.form customization</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="mode">extension</field>
+        <field name="model">res.partner</field>
+        <field name="priority">160</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='vat']" position="after">
+                    <field name="x_condominiums_ids" invisible="False" widget="many2many_tags" options="{&quot;no_create&quot;:true}" force_save="1" readonly="True"/>
```

Comment:
- Field is already readonly
- By default the field is visible
- Prefer python in options
- Avoid `&quot;` and others if useless
```suggestion
                    <field name="x_condominiums_ids" widget="many2many_tags" options="{'no_create': True}" force_save="1"/>
```